docs/tutorial/gatsby-image-tutorial/gatsby-image-tutorial.md

```diff
@@ -0,0 +1,361 @@
+
+```bash
+npm install gatsby-source-filesystem
+```
+
+2. The next step is to make sure your `gatsby-config.js` file specifies the correct folder. In this example it would look like this:
+
+```javascript:title=gatsby-config.js
+plugins: [
+  `gatsby-transformer-sharp`,
+  `gatsby-plugin-sharp`,
+  { resolve: `gatsby-source-filesystem`, options: { path: `./src/data/` },
+]
+```
+
+Now you're ready to start working with `gatsby-image`!
+
+## Step 4
+
+The next step can vary depending on what you're trying to accomplish.
+
+## Querying data for a single image
+
+Use `graphql` to query an image file directly. You can include the relative path to the image file and determine how you want `gatsby-image` to process the file.
+
+```jsx:title=src/pages/index.js
+export const query = graphql`
+  query {
+    file(relativePath: { eq: "headers/headshot.jpg" }) {
+      childImageSharp {
+        fixed(width: 125, height: 125) {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
+```
+
+There are a couple of things to note here.
+
+### Relative image paths and `gatsby-config.js`
+
+You might expect the relative path to be relative to the file the code sits in, in this case that's index.js. However, that doesn't work. The relative path is actually based on the line of code you put in the `gatsby-source-filesystem` config, which points to `src/data`.
+
+### Image fragments
+
+Another thing to note about this query is how it uses the fragment `GatsbyImageSharpFixed` to return a fixed width and height image. You could also use the fragment `GatsbyImageSharpFluid` which produces scalable images that fill their container instead of fitting specific dimensions. In `gatsby-image`, _fluid_ images are meant for images that don’t have a finite size depending on the screen, where as other images are _fixed_.
+
+The query will return a processed image at the highest depth. That is, the deepest part of the return object. The returned result will be automatically passed into the component and attached to the `data` prop. You can then display the image using JSX to automatically output responsive, highly performant HTML.
+
+To display the image, start by importing the component provided by `gatsby-image`.
+
+```jsx
+import Img from "gatsby-image"
+```
+
+Now you can use it. Note that the key for pointing to the image corresponds to the way in which the image was processed. In this example that is `fixed`.
+
+```jsx
+<Img
+  className="headshot"
+  fixed={data.file.childImageSharp.fixed}
+  alt="headshot"
+/>
+```
+
+Here is the query and usage all put together:
+
+```jsx:title=src/pages/index.js
+import React from "react"
+import { graphql } from "gatsby"
+import Img from "gatsby-image"
+import Layout from "../components/layout"
+
+const HomePage = ({ data }) => {
+  return (
+    <Layout>
+      <Img
+        className="headshot"
+        fixed={data.file.childImageSharp.fixed}
+        alt="headshot"
```

Comment:
```suggestion
        alt=""
```
"headshot" wouldn't be super useful for users of assistive technology, so I'd recommend leaving it blank or pulling in the author/site owner's name.